util/secscan/v4/api.py

```diff
@@ -196,6 +206,15 @@ def _perform(self, action):
 
         return resp
 
+    def _sign_jwt(self):
```

Comment:
In this function, It appears there's two primary paths:
```
if self.jwt_psk
```
and
```
else
```
For readability purposes, it might make more sense to wrap everything that should happen when `self.jwt_psk` evaluates to true/exists, and put the alternate/fallback into an `else` statement.